bootstrap.sh

```diff
@@ -19,4 +19,4 @@ sudo -u postgres psql -c "ALTER DATABASE ppe_dev OWNER TO ppe"
 
 echo -e "************     Installing Dependencies     *************"
 cd ../../vagrant
```

Comment:
What is the absolute path of _vagrant_ dir? /home/vagrant? If so, you could change it to something like:

> cd ~vagrant